docs/usage.md

```diff
@@ -104,9 +105,32 @@ class MyLimitPagination(JsonApiLimitOffsetPagination):
 
 ### Filter Backends
 
-Following are descriptions for two JSON:API-specific filter backends and documentation on suggested usage
+Following are descriptions for three JSON:API-specific filter backends and documentation on suggested usage
```

Comment:
just a detail but referencing the number of filters here sounds a bit like a overkill and we might forget changing it in the future when we add more.

Why not just:
`Following are descriptions of JSON:API-specific filter backends and documentation on suggested usage`